pkgdb2/lib/__init__.py

```diff
@@ -1972,15 +1975,21 @@ def add_new_package_request(
     if pkg_collection.startswith(('el', 'epel')):
         _validate_pkg(session, pkg_collection[-1:], pkg_name)
 
+    if pkg_description:
```

Comment:
I do not follow. Before my change pkgdb would use the empty string if pkg_description is just a  space character:
```
In [3]: pkg_description = " "

In [4]: pkg_description.strip() if pkg_description else None
Out[4]: ''

```